flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/MySqlToMySqlE2eITCase.java

```diff
@@ -0,0 +1,234 @@
+                    throw new RuntimeException("MYSQL_SINK startup timed out.");
+                }
+                LOG.info("Waiting for MYSQL_SINK to be available");
+                Thread.sleep(1000);
+            } catch (InterruptedException ignored) {
+                // ignore and check next round
+            }
+        }
+        LOG.info("Containers are started.");
+    }
+
+    @Before
+    public void before() throws Exception {
+        super.before();
+        MYSQL_SOURCE.createDatabase(TEST_DB);
+        MYSQL_SOURCE.createAndInitialize("mysql_inventory");
+        MYSQL_SINK.createDatabase(TEST_DB);
+        jdbcJar = TestUtils.getResource(getJdbcConnectorResourceName());
+        // MYSQL_SINK.createAndInitialize("mysql_inventory");
+    }
+
+    protected String getJdbcConnectorResourceName() {
+        return String.format("jdbc-connector_%s.jar", flinkVersion);
+    }
+
+    @After
+    public void after() {
+        super.after();
+        MYSQL_SOURCE.dropDatabase(TEST_DB);
+        MYSQL_SINK.dropDatabase(TEST_DB);
+    }
+
+    @Test
+    public void test() throws Exception {
+        String pipelineJob =
+                String.format(
+                        "source:\n"
+                                + "  type: mysql\n"
+                                + "  hostname: "
+                                + SOURCE_NETWORK_NAME
+                                + "\n"
+                                + "  port: 3306 \n"
+                                + "  username: %s\n"
+                                + "  password: %s\n"
+                                + "  tables: %s.\\.*\n"
+                                + "  server-id: 5400-5404\n"
+                                + "  server-time-zone: UTC\n"
+                                + "  jdbc.properties.useSSL: false\n"
+                                + "  jdbc.properties.allowPublicKeyRetrieval: true\n"
+                                + "\n"
+                                + "sink:\n"
+                                + "  type: mysql-writer\n"
+                                + "  hostname: "
+                                + SINK_NETWORK_NAME
+                                + "\n"
+                                + "  port: 3306 \n"
+                                + "  username: %s\n"
+                                + "  password: %s\n"
+                                + "\n"
+                                + "pipeline:\n"
+                                + "  parallelism: 1",
+                        TEST_USERNAME,
+                        TEST_PASSWORD,
+                        TEST_DB,
+                        TEST_USERNAME,
+                        TEST_PASSWORD);
+
+        System.out.println(pipelineJob);
+
+        Path mysqlCdcJar = TestUtils.getResource("mysql-cdc-pipeline-connector.jar");
+        Path jdbcCdcJar = TestUtils.getResource("jdbc-cdc-pipeline-connector.jar");
+        Path hikariJar = TestUtils.getResource("HikariCP.jar");
```

Comment:
We've included this HikariCP dependency into mysql-cdc-pipeline-connector.jar package or jdbc-cdc-pipeline-connector.jar, so we don't need to upload this jar?